### PR TITLE
EXLM-83: accordion block

### PR DESCRIPTION
**Pull request by gandal-adobe**
Please always provide the Jira Issue your PR is for, as well as test URLs where your change can be observed (before and after):

Jira ID: EXLM-83

Test URLs:

- Before: https://main--exlm--adobe-experience-league.hlx.live/docs/authoring-guide-exl/test/collapsible-content
- After: https://feature-exlm-83-accordion--exlm--adobe-experience-league.hlx.live/docs/authoring-guide-exl/test/collapsible-content

### CODE REVIEW

File: blocks/accordion/accordion.scss [hunk 1]
```diff
@@ -0,0 +1,6 @@
+.accordion {
+  background-color: var(--spectrum-gray-100);
+  padding: 0 16px;
+  padding-top: 16px;
+  padding-bottom: 16px;
```
Comment (thomsebastin) [inline blocks/accordion/accordion.scss:5]: This could just be `padding: 16px`

File: blocks/accordion/accordion.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+    const summaryElement = document.createElement('summary');
+    const summaryText = block.querySelector('div > div:first-child').firstElementChild.innerText;
+    summaryElement.innerText = summaryText;
+
+    const detailsElement = document.createElement('details');
+    const pElement = document.createElement('p');
+    block.removeChild(block.querySelector('div > div:first-child'));
+
+    pElement.innerHTML = block.innerHTML;
```
Comment (ahmed-musallam) [inline blocks/accordion/accordion.js:11]: Block content should not live in a `<p>`, rather it should be copied into `details` as-is

**Review by ahmed-musallam — CHANGES_REQUESTED** [review]

File: blocks/accordion/accordion.js (hunk 2 above)
Comment (gandal-adobe) [reply blocks/accordion/accordion.js]: modified to:
```
    // next div element(s) form the accordion details
    Array.from(block.children).forEach((element) => {
        detailsElement.appendChild(element);
    });
```

**Comment by ahmed-musallam** [comment]
@gandal-adobe LGTM

**Review by ahmed-musallam — APPROVED** [review]

**Review by thomsebastin — APPROVED** [review]

**Pull request merged**